js/app/application.js

```diff
@@ -91,9 +91,12 @@ const Application = new Knowledge.Class({
                              'Path to a different gresource to use with the application', null);
         this.add_main_option('theme-overrides-path', 'O'.charCodeAt(), GLib.OptionFlags.NONE, GLib.OptionArg.FILENAME,
                              'Path to a overrides scss or css file to theme the application', null);
+        this.add_main_option('web-overrides-path', 'w'.charCodeAt(),
+            GLib.OptionFlags.NONE, GLib.OptionArg.FILENAME,
```

Comment:
The option flags are also on the same line as `add_main_option()` for the other command line arguments.